test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -54,6 +57,12 @@ def send_message(self, message: str):
         type(ChatComponents.MESSAGE_INPUT.value, message)
         press_enter(ChatComponents.MESSAGE_INPUT.value)
         
+    def type_message(self, message: str):
+        type(ChatComponents.MESSAGE_INPUT.value, message)
```

Comment:
We do this in multiple places in this file, might as well use this nice new function.